### ISSUE TITLE
Having title and filename is annoying.

### ISSUE THREAD

**Issue by Omay238** [hackclub/sprig#474]
We should: remove filename - OR - use title ONLY for the title, not files.

**Comment by leomcelroy** [comment]
Yes, let's only have the file name. Title can come from the file.

**Comment by Omay238** [comment]
@leomcelroy i fixed it in [this](https://github.com/hackclub/sprig-gallery/pull/34) pr

**Comment by alhardwarehyde** [comment]
Closing because it's fixed! It was fixed in the V1 Sprig UI and is also fixed in the New UI we're shipping soon!
